src/EditorFeatures/Core.Wpf/InlineRename/AbstractEditorInlineRenameService.SymbolRenameInfo.cs

```diff
@@ -53,6 +54,7 @@ private partial class SymbolInlineRenameInfo : IInlineRenameInfo
                 TextSpan triggerSpan,
                 SymbolAndProjectId renameSymbolAndProjectId,
                 bool forceRenameOverloads,
+                bool isCaseSensitiveAttribute,
```

Comment:
I know we use this flag for handling Attributes but the value represents whether the language as a whole is case sensitive. Perhaps a name like `isLanguageCaseSensitive` might be more descriptive.